Misc/NEWS.d/next/Tests/2019-05-04-21-25-19.bpo-36782.h3oPIb.rst

```diff
@@ -0,0 +1 @@
+Created tests for several C API functions in the datetime module, specifically::
```

Comment:
just remove `::`

Reply:
maybe this sentence `Add tests for several C API functions in the :mod:`datetime` module. Patch by Edison Abahurire`